packages/components/psammead-consent-banner/README.md

```diff
@@ -21,6 +21,7 @@ The `psammead-consent-banner` component is a styled `div` that encapsulates info
 | reject | element | Yes | N/A | `<a href="https://www.bbc.co.uk/usingthebbc/your-data-matters">Find out what's changed</a>` |
 | id | string | No | Null | `ConsentBanner` |
 | hidden | bool | No | Null | `false` |
+| dir | string | Yes | N/A | One of `'rtl'` `'ltr'` |
```

Comment:
Are we sure we don't want to default to `ltr`?

Reply:
Updated to default to ltr.